<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of Images:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QLabel" name="numImagesLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTextBrowser" name="output"/>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
